core/src/gql/franchise/franchise.object.ts

```diff
@@ -2,15 +2,15 @@ import { Field, ObjectType } from '@nestjs/graphql';
 
 @ObjectType('Franchise')
 export class FranchiseObject {
-  @Field()
-  id: string;
+	@Field()
+	id: string;
 
-  @Field()
-  createdAt: Date;
+	@Field()
+	createdAt: Date;
```

Comment:
Another comment that is very addressable in a separate PR. Is it not possible for us to have a BaseObject like we do with BaseEntity?